docs/getting_started.md

```diff
@@ -46,10 +46,22 @@ In order for `clusterctl` to bootstrap a management cluster on vSphere, it must
 
 #### Uploading the CAPV Machine Image
 
-It is required that machines provisioned by CAPV use one of the official CAPV machine images as a VM template. The machine images are retrievable from public URLs. CAPV currently supports machine images based on Ubuntu 18.04 and CentOS 7. A list of published machine images is available [here](../README.md#kubernetes-versions-with-published-ovas). For this guide we'll be deploying Kubernetes v1.16.2 on Ubuntu 18.04 (link to [machine image](https://storage.googleapis.com/capv-images/release/v1.16.2/ubuntu-1804-kube-v1.16.2.ova)).
+It is required that machines provisioned by CAPV use one of the official CAPV machine images as a VM template. The machine images are retrievable from public URLs. CAPV currently supports machine images based on Ubuntu 18.04 and CentOS 7. 
+- A list of published machine images is available [here](../README.md#kubernetes-versions-with-published-ovas) you can pick one. For this guide we'll be deploying Kubernetes v1.16.2 on Ubuntu 18.04 (link to [machine image](https://storage.googleapis.com/capv-images/release/v1.16.2/ubuntu-1804-kube-v1.16.2.ova)).
 
 [Create a VM template](https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-17BEDA21-43F6-41F4-8FB2-E01D275FE9B4.html) using the OVA URL above. The rest of the guide will assume you named the VM template `ubuntu-1804-kube-v1.15.4`.
 
+##### Quick Example for Ubuntu which enables Linked clones
+
+There are many ways to do this, but here is an example workflow for Ubuntu18 which should just work for your first time installation: 
+
+- click on your ESXI parent, and click "Deploy OVF Template"
+- Enter http://storage.googleapis.com/capv-images/release/v1.16.3/ubuntu-1804-kube-v1.16.3.ova as the value of the URL for this template.
```

Comment:
The example above uses 1.16.2 but we're using 16.3 here, we should be consistent and use the same version (prefer 1.16.3)

Reply:
same here, https, I don't think storage.googleapis.com does a https redirect